VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/vho-chat/vho-chat.component.ts

```diff
@@ -52,6 +55,10 @@ export class VhoChatComponent extends ChatBaseComponent implements OnInit, OnDes
     }
 
     sendMessage() {
+        console.log(this.newMessageBody);
```

Comment:
console.log not logger ?

Reply:
needs to be removed. was for testing purposes.